packages/react/src/Testimonial/Testimonial.tsx

```diff
@@ -170,7 +170,7 @@ function _Logo({children, ...rest}: LogoProps, ref) {
   )
 }
 
-export const Logo = forwardRef(_Logo)
+const Logo = forwardRef(_Logo)
```

Comment:
Nice! Thanks for removing these! Can we also remove the custom Avatar from this file altogether?

Reply:
Absolutely! It's updated now.